src/excel/Workbook.js

```diff
@@ -23,65 +23,79 @@ export class Workbook extends NamedItemContainer {
     this._log = log;
   }
 
-  async getData() {
-    const result = await this._oneDrive.doFetch(this._uri);
-    return result.value;
-  }
-
-  async getWorksheetNames() {
-    this.log.debug(`get worksheet names from ${this._uri}/worksheets`);
-    const result = await this._oneDrive.doFetch(`${this._uri}/worksheets`);
-    return result.value.map((v) => v.name);
-  }
-
-  worksheet(name) {
-    return new Worksheet(this._oneDrive, `${this._uri}/worksheets`, name, this._log);
-  }
-
   async createSession() {
```

Comment:
this is a bit ugly... I would throw an exception if a session id already exists and add a `getSessionId()`